libs/SalesforceSDK/res/values/bootconfig.xml

```diff
@@ -7,4 +7,5 @@
         <item>api</item>
         <item>web</item>
     </string-array>
+    <string name="remoteAccessConsumerSecret"></string>
```

Comment:
Hmm, I thought bundling the consumer secret in a mobile app was a no-no for security.